app/src/main/res/values/strings.xml

```diff
@@ -25,4 +26,7 @@
     <string name="TextShouldNotBeBlank">"Text should not be blank"</string>
     <string name="TextIsTooLong">"Text is too long"</string>
     <string name="TextIsValid">""</string>
+    <string name="FabAddIcon">Fab add icon</string>
+    <string name="PartyTextFieldTitle">Party Title</string>
+    <string name="PartyTextFieldDescription">Description</string>
 </resources>
```

Comment:
names are to generic here, it should indicate more the place and usage :)

Reply:
Updated